llm_compmat/llm.py

```diff
@@ -6,11 +6,26 @@
 )
 from langchain.agents.output_parsers.openai_tools import OpenAIToolsAgentOutputParser
 
+
+SYSTEM_PROMPT = """
+You are very powerful assistant, but don't know current events.
+You can calculate materials properties with EMT and MACE.
```

Comment:
I was thinking that the description of the methods should go in the doc string of the tool that use them. So that we keep the system prompt more general. I did some tests previously and saw that the llm will know about the docstrings of the tools, so it should work as well.